### PR TITLE
#1058 Brick detail. Styles.

### PR DESCRIPTION
**Pull request by BALEHOK**
Fixing a couple of styles issues plus applying CSS modules.
Let's review the approach to styling in this PR (related to #985 ).

cc @BLoe

### CODE REVIEW

File: src/options/pages/brickEditor/BlockResult.module.scss [hunk 1]
```diff
@@ -15,6 +15,24 @@
...
+  cursor: pointer;
+  border-radius: 0;
+  text-align: left;
+
+  &:hover {
+    color: #fff;
+    background-color: #b66dff;
+  }
+
+  &:global(.list-group-item).active {
```
Comment (BALEHOK) [inline src/options/pages/brickEditor/BlockResult.module.scss:29]: Have to use more powerful selector to override default `.list-group-item.active` styles.

File: src/options/pages/brickEditor/BlockResult.module.scss [hunk 2]
```diff
@@ -15,6 +15,24 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+.root {
```
Comment (BALEHOK) [inline src/options/pages/brickEditor/BlockResult.module.scss:18]: If the topmost element of a component has a style, it always has the name `.root`. Just for the sake of consistency.

File: src/options/pages/brickEditor/BlockResult.module.scss (hunk 1 above)
Comment (twschiller) [reply src/options/pages/brickEditor/BlockResult.module.scss:29]: For PR comments like these, it's helpful to put them in the code itself so it's there for future readers

**Comment by twschiller** [comment]
So many scrolls! Not sure any way around it at this point. Something to keep in the back of our mind. Onward and upward!

image

**Pull request merged**